backend/main/models/vote.go

```diff
@@ -194,13 +200,36 @@ func (v *Vote) GetVoteById(db *s.Database) error {
 }
 
 func (v *Vote) CreateVote(db *s.Database) error {
+	var defaultEarlyVoteLength = 1
+
+	// Create Vote
 	err := db.Conn.QueryRow(db.Context,
 		`
 		INSERT INTO votes(proposal_id, addr, choice, composite_signatures, cid, message)
 		VALUES($1, $2, $3, $4, $5, $6)
 		RETURNING id, created_at
 	`, v.Proposal_id, v.Addr, v.Choice, v.Composite_signatures, v.Cid, v.Message).Scan(&v.ID, &v.Created_at)
 
+	// Get Proposal Start time to check for early vote
+	var proposal Proposal
+	pgxscan.Get(db.Context, db.Conn, &proposal,
+		`SELECT start_time, community_id from proposals
+		WHERE id = $1`,
+		v.Proposal_id)
+
+	isEarlyVote := proposal.Start_time.Before(proposal.Start_time.Add(time.Hour + time.Duration(defaultEarlyVoteLength)))
```

Comment:
Maybe i'm reading this wrong but i think this would always evaluate as true. Should this compare if the current time is less than 1 hour from the proposal start time?

Reply:
Yep - good catch. That is supposed to be checking `v.Created_at` - which is when the user voted and comparing if that is before the early vote cut off on the proposal.